tests/test_rendering.py

```
@@ -1,46 +1,138 @@
+        ("white", (4 * SQUARE_SIZE, 4 * SQUARE_SIZE)),
+        ("black", (3 * SQUARE_SIZE, 3 * SQUARE_SIZE)),
+    ],
+)
+def test_board_svg_places_piece_on_expected_square(orientation, square_origin):
+    status, content_type, svg_data = get_response(
+        request_url("/board.svg", fen=PAWN_FEN, size=BOARD_SIZE, orientation=orientation)
     )
 
+    assert status == 200
+    assert content_type == "image/svg+xml"
+    root = ElementTree.fromstring(svg_data)
```

Ruff (0.16.1)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)